x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/action/bulk/TransportBulkShardOperationsAction.java

```diff
@@ -180,12 +180,11 @@ public static WritePrimaryResult<BulkShardOperationsRequest, BulkShardOperations
                 appliedOperations.add(targetOp);
                 location = locationToSync(location, result.getTranslogLocation());
             } else {
-                if (result.getFailure() instanceof AlreadyProcessedFollowingEngineException) {
+                if (result.getFailure()instanceof final AlreadyProcessedFollowingEngineException failure) {
```

Comment:
Space before `instanceof` please

Reply:
I wonder if this is how it is enforced by the spotless?